### PR TITLE
qu-auto/SORQA-475 Automatize "Make region and district required for aggregate reports [0.5]"

### PR DESCRIPTION
**Pull request by hms-sgent**
Fixes #

### CODE REVIEW

**Comment by hms-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1634/
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1637/
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1638/

**Comment by hms-sgent** [comment]
image

**Review by lb-sgent — APPROVED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java [hunk 1]
```diff
@@ -258,6 +258,8 @@ public TaskManagementSteps(
     When(
         "I check if popup message is {string}",
         (String expectedText) -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(
+              By.cssSelector(".v-Notification-description"));
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java:262]: create locator in page object for notification popup

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java [hunk 2]
```diff
@@ -149,5 +155,39 @@ public MSersDirectorySteps(
...
+              FIRST_AGGREGATED_REPORT_EDIT_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(FIRST_AGGREGATED_REPORT_EDIT_BUTTON);
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(EDIT_AGGREGATED_REPORT_HEADER);
+        });
+
+    And(
+        "^I check that first row contains \"([^\"]*)\" in column (\\d+)$",
+        (String expectedResult, Integer columnNumber) -> {
+          String actualResult =
+              webDriverHelpers.getTextFromWebElement(
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java:182]: why new locator here?

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/CreateNewAggregateReportSteps.java [hunk 3]
```diff
@@ -123,6 +125,19 @@ public CreateNewAggregateReportSteps(
...
         });
+    And(
+        "^I set (\\d+) as the quantity for Snake Bite suspected cases in Create a new aggregated report$",
+        (Integer quantity) -> {
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(SNAKE_BITE_SUSPECTED_CASES_INPUT);
+          webDriverHelpers.fillInWebElement(
+              SNAKE_BITE_SUSPECTED_CASES_INPUT, String.valueOf(quantity));
+        });
+    When(
+        "^I close popup message window$",
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/CreateNewAggregateReportSteps.java:136]: specify location of this popup

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java (hunk 1 above)
Comment (hms-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java:262]: done

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/CreateNewAggregateReportSteps.java (hunk 3 above)
Comment (hms-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/CreateNewAggregateReportSteps.java:136]: done

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java (hunk 2 above)
Comment (hms-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/mSers/MSersDirectorySteps.java:182]: getElementFromFirstRowAndSpecificColumnByColumnNumber created and moved to MSersDirectoryPage.java

**Comment by hms-sgent** [comment]
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1675/allure/
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1676/allure/
https://jenkins.sormas.netzlink.com/job/sormas-RunE2eTests-branch/1677/allure/

**Pull request merged**